### ISSUE TITLE
lion records

### ISSUE THREAD

**Issue by klg-2016** [kaitlyngaynor/gorongosa-mesocarnivores#135]
next task I'm working on is calculating number of lion records during our study period, and I cannot remember how we did that for the focal species from my masters thesis. I'm going to take some time now to remind myself, but I'm posting this in case you see it and it's a really simple reminder to guide my working on this.

**Comment by kaitlyngaynor** [comment]
It's in the file I just sent you! add up the PALE column = 9

**Comment by kaitlyngaynor** [comment]
in detections-dry2016_katie-8_1_16-11_30_16.csv (you can get # of detections for all species)

**Comment by kaitlyngaynor** [comment]
Or maybe that's the number of camera-days with lion detections, but for these purposes, that's probably sufficient to report.

**Comment by klg-2016** [comment]
Thank you! I just checked for civets, and the two values are different - the sum from that table for civets is 407 (so number of camera-days with civet detections) and what I reported in my thesis was 612 detections (I just don't remember how we calculated those values). I can filter the record table to the correct dates and only lions, then just manually count the number of detections (using the 30 second rule) and I get 12, which seems to make sense if there were 9 days with lions detected.
